### ISSUE TITLE
telomerase difference

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12633]
I'm checking a community submission related to telomerase.

I don't understand the difference between these 2 terms:

GO:0003720 telomerase activity
Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1). Catalyzes extension of the 3'- end of a DNA strand by one deoxynucleotide at a time using an internal RNA template that encodes the telomeric repeat sequence.

telomerase RNA reverse transcriptase activity
Catalysis of the extension of the 3' end of a DNA strand by one deoxynucleotide at a time. Cannot initiate a chain de novo; uses the RNA subunit of the telomerase enzyme complex as its template.

**Comment by ValWood** [comment]
GO:0032203	telomere formation via telomerase
A cellular process that results in the formation of a telomere at a non-telomeric double-stranded DNA end that involves the activity of a telomerase enzyme. 

So, I presume this is describing elongation (although I am a little confused by the  meaning of the 'telomere assembly' term?

I expected to see a relationship here


![relationship](https://cloud.githubusercontent.com/assets/7359272/18225274/efd5cdf4-71e6-11e6-9eee-abceda32e0f3.jpg)

**Comment by RLovering** [comment]
Hi Val, Nancy is back in October, do you mind waiting for her input? 
Ruth

**Comment by RLovering** [comment]
@NancyCampbell

**Comment by ValWood** [comment]
No hurry,  the session is curated using 
"positive regulation of telomere maintenance via telomerase"
https://curation.pombase.org/pombe/curs/be9521cb5c4791ae
but the parentage doesn't seem to encapsulate the process of DNA biosynthesis in any way as it stands.
@mah11  may also have input as she has been curating in this area, and may have some knowledge of the current arrangement. No hurry though.

**Comment by mah11** [comment]
> I don't understand the difference between these 2 terms:

Telomerase activity (GO:0003720) has two parts: the reverse transcriptase activity (GO:0003721) and the RNA template "activity" (GO:0000332). Being a template isn't as graceful a fit with GO MF's "molecular activity" scope as your typical catalytic activity, but it is what the telomerase RNA is there for, and people have wanted to annotate its role at the molecular level. If it's done at all, GO is doing it as best it can.

> GO:0032203 telomere formation via telomerase
> [should it link to GO:0022616 DNA strand elongation?]

There isn't a link, and it looks like there should be, but it will have to be has_part (telomere formation via telomerase has_part DNA strand elongation). I think QuickGO doesn't show has_part links, so its display would look the same with the link in the ontology.

> My question is then, which term to use for "telomere elongation by telomerase"

I would say GO:0032203 for the process, and GO:0003720 if you're actually annotating the telomerase itself.

**Comment by ValWood** [comment]
Hmm, making the has_part link won't really prevent annotation inconsistence though...

![inconsistency](https://cloud.githubusercontent.com/assets/7359272/18249356/a3aadcc2-7375-11e6-9c27-0ddfc8513a6b.jpg)

On telomerase catalytic subunit, the same process can be correctly described in 2 ways, either as 'telomere maintenance',  or as 'DNA elongation'

**Comment by ValWood** [comment]
I don't understand why the process
"telomere maintenance by telomerase"
can't be equally called 
"DNA synthesis by telomerase" (or exact synonym) as this is what telomerase is doing

and as telomerase activity is defined as 
"Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1). Catalyzes extension of the 3'- end of a DNA strand by one deoxynucleotide at a time using an internal RNA template that encodes the telomeric repeat sequence."

The term "telomere maintenance by telomerase/DNA synthesis by telomerase" could be "part_of" the broader telomere maintenance term

**Comment by tberardini** [comment]
@NancyCampbell :  back in September, Ruth mentioned that you'd be  back in October, do you mind waiting for her input?"  Could you please have a look at this issue?  Thanks.

**Comment by NancyCampbell** [comment]
My two pennies' worth:

Telomerase is an enzyme formed of two parts TERT(protein) and TERC (RNA template). So neither can have/perform telomerase activity without the other,

Therefore I would expect to see:
TERT contributes_to Telomerase activity GO:0003720
TERC contributes_to Telomerase activity GO:0003720
Then
TERT-TERC complex annotated directly (no contribues_to) to Telomerase activity GO:0003720
Furthermore,
TERC annotated to telomerase RNA reverse transcriptase activity GO:0003721

Having said that:
That is not how the annotations on QuickGO currently stand. Depending on what is agreed here, some work might need to be done. I can do ours to make sure they are standardised. Any comments?

**Comment by NancyCampbell** [comment]
and definitely Val ...then the definitions will need some adjusting! I agree they are not very clear as they stand now!

**Comment by NancyCampbell** [comment]
The links may possibly go further down the tree to the children?!
![screen shot 2016-10-31 at 10 37 01](https://cloud.githubusercontent.com/assets/12081136/19851704/30d3d4ca-9f57-11e6-9cb0-ae7f1256acb7.png)

**Comment by ValWood** [comment]
@NancyCampbell 
could you summarize the reationships to be added?
(I.e what the red lines represent...I'm not sure what needs to be added.
Val

(I have another question...isn't DNA strand elongation always a DNA biosynthetic process?)

**Comment by NancyCampbell** [comment]
Hi @ValWood 

How does the following sound:
telomerase RNA reverse transcriptase activity part_of telomeric 3' overhang formation part_of telomere formation via telomerase.

What do the editors @tberardini think about this? should this type of relationship at this granular level be relfected in the ontology tree itself? - the curator can add it as part of the annotation itself in the AE field?!

Val, I am surprised though that telomere formation via telomerase has no children?? What am i missing?

As to the question: (I have another question...isn't DNA strand elongation always a DNA biosynthetic process?)

I think it is!! the definitions on QuickGO below are in agreement too! But it is not reflected in the tree (below)!!! 

**DNA biosynthetic process:** GO:0071897 The cellular DNA metabolic process resulting in the formation of DNA, deoxyribonucleic acid, one of the two main types of nucleic acid, consisting of a long unbranched macromolecule formed from one or two strands of linked deoxyribonucleotides, the 3'-phosphate group of each constituent deoxyribonucleotide being joined in 3',5'-phosphodiester linkage to the 5'-hydroxyl group of the deoxyribose moiety of the next one. 

**DNA strand elongation:** GO:0022616 The DNA metabolic process in which a DNA strand is synthesized by adding nucleotides to the 3' end of an existing DNA stand.

The parent of both of the terms above is  GO:0006259 **DNA metabolic process**: Any cellular metabolic process involving deoxyribonucleic acid. This is one of the two main types of nucleic acid, consisting of a long, unbranched macromolecule formed from one, or more commonly, two, strands of linked deoxyribonucleotides

![screen shot 2016-11-07 at 10 14 09](https://cloud.githubusercontent.com/assets/12081136/20054059/21ed96ae-a4d3-11e6-8bb3-1391c74eac91.png)

**Comment by mah11** [comment]
Various comments, which I hope don't add to the confusion ...

Annotations:

> TERT-TERC complex annotated directly (no contribues_to) to Telomerase activity GO:0003720
> TERC annotated to telomerase RNA reverse transcriptase activity GO:0003721

Should the second annotations say TERT (not TERC)? If so, these both look fine. Furthermore, annotating TERT to GO:0003721 makes it unnecessary to include this one explicitly:

> TERT contributes_to Telomerase activity GO:0003720

... because the ontology has GO:0003721 part_of GO:0003720. Similarly, 'template for synthesis of G-rich strand of telomere DNA activity' GO:0000332 is also part_of GO:0003720, so if TERC is annotated to GO:0000332 it doesn't need to be annotated to GO:0003720 as well.


Ontology:

I'm no expert on telomere biology, so I don't know whether what GO has is exactly as it should be, especially in BP.

> telomerase RNA reverse transcriptase activity part_of telomeric 3' overhang formation (GO:0031860)

Does telomerase only synthesize the 3' overhang? That's what this link would imply (i.e. the part_of link won't always be correct if telomerase also synthesizes any of the telomere other than the overhang).

> telomeric 3' overhang formation part_of telomere formation via telomerase

This may well be true - it's one of the things I don't know so well. In any case, the GO:0031860 definition could probably stand to be improved, to say more explicitly what it entails (its current shortcomings are probably my fault; the term dates back to before any of us were so careful about beginnings, ends, and MF parts of processes).

> I don't understand why the process "telomere maintenance by telomerase" can't be equally called "DNA synthesis by telomerase" 

Again, not sure but it may just be to make the "telomere maintenance" term names match nicely.

I seem to recall people (Eurie and Rob? guessing from def dbxrefs) saying that telomeres could be maintained either by synthesis or by recombination, which accounts for GO:0007004, GO:0032201 and GO:0000722. I also wonder if GO:0032201 should be is_a  telomere maintenance via telomere lengthening GO:0010833?

> isn't DNA strand elongation always a DNA biosynthetic process?

I think the strand elongation defs in GO are a bit simplistic; I would say that strand elongation is a process that always has DNA biosynthesis as a part, but not necessarily the only part. Certainly in the context of replication, lagging strand elongation involves primer synthesis (i.e. an RNA biosynthetic process) as well as DNA synthesis, and for both strands helicase and ligase activities are necessary, and not obviously covered by the GO:0022616 def. I would adjust the elongation defs to include the other activities, and then say elongation has_part synthesis.

**Comment by tberardini** [comment]
Great discussion. When it all settles down, please enumerate the changes you would like to see and I will implement.

@NancyCampbell : Can you explain this one please, 
>"telomerase RNA reverse transcriptase activity part_of telomeric 3' overhang formation part_of telomere formation via telomerase."?

Does this mean:

telomere formation via telomerase.
--[p] telomeric 3' overhang formation
---[p] telomerase RNA reverse transcriptase activity

>should this type of relationship at this granular level be relfected in the ontology tree itself? 

Sure, if that's a true capture of the biology.

**Comment by NancyCampbell** [comment]
_Comments in italics below (I hope it works)_

Various comments, which I hope don't add to the confusion ...

_On the contrary, clear. Thank you_

Annotations:

    TERT-TERC complex annotated directly (no contribues_to) to Telomerase activity GO:0003720
    TERC annotated to telomerase RNA reverse transcriptase activity GO:0003721

Should the second annotations say TERT (not TERC)? If so, these both look fine. Furthermore, annotating TERT to GO:0003721 makes it unnecessary to include this one explicitly:

_Yes (typo- sorry!)
    TERT annotated to telomerase RNA reverse transcriptase activity GO:0003721
&
    TERC annotated to template for synthesis of G-rich strand of telomere DNA activity GO:0000332 (which you mention/allude to in the next point - unless I am mistaken)_

    TERT contributes_to Telomerase activity GO:0003720

... because the ontology has GO:0003721 part_of GO:0003720. Similarly, 'template for synthesis of G-rich strand of telomere DNA activity' GO:0000332 is also part_of GO:0003720, so if TERC is annotated to GO:0000332 it doesn't need to be annotated to GO:0003720 as well.

_ yes 
so is this what the annotation looks like (?):
TERT annotated to telomerase RNA reverse transcriptase activity GO:0003721
TERC annotated to template for synthesis of G-rich strand of telomere DNA activity GO:0000332
TERT-TERC complex annotated to Telomerase activity GO:0003720

if yes, and since GO:0003721 & GO:0000332 are children of GO:0003720 and consequently do not need to be annotated explicitly to GO:0003720 (which is the sensible/usual thing to do as suggested), then my question would be:

how can one indicate that both components are needed together for telomerase activity and neither of them can perform the activity alone, in context of annotating an exisiting complex term?  

suggestion 1: an automatic mechanism that will inherit/cascade this annotation to the components of the complex from the annotation of TERT-TERC complex to telomerase activity GO:0003720, does this even exist? (I feel it was one of the options in discussion amongst other alternatives on how to deal with this in the past but I am not aware of the outcome - was a decision made and implemented? And in the meantime how do we deal with it?)

Suggestion 2:

TERT contributes_to Telomerase activity GO:0003720
TERC contributes_to Telomerase activity GO:0003720

The emphasis above is on using the qualifier contributes_to
_

Ontology:

I'm no expert on telomere biology, so I don't know whether what GO has is exactly as it should be, especially in BP.

    telomerase RNA reverse transcriptase activity part_of telomeric 3' overhang formation (GO:0031860)

Does telomerase only synthesize the 3' overhang? That's what this link would imply (i.e. the part_of link won't always be correct if telomerase also synthesizes any of the telomere other than the overhang).

_so if telomerase has other functions, for example some are TERC-independent i.e. TERT binds with RMRP and the TERT-RMRP complex has TERC-INDEPENDENT RNA-DEPENDENT RNA POLYMERASE ACTIVITY PMID 23061047 (figure 1 below) (review), the link can't be made, correct?

![fonc-02-00133-g001](https://cloud.githubusercontent.com/assets/12081136/20136986/b8a215aa-a66f-11e6-9af3-75c713713bde.jpg)
 _

    telomeric 3' overhang formation part_of telomere formation via telomerase

This may well be true - it's one of the things I don't know so well. In any case, the GO:0031860 definition could probably stand to be improved, to say more explicitly what it entails (its current shortcomings are probably my fault; the term dates back to before any of us were so careful about beginnings, ends, and MF parts of processes).

    I don't understand why the process "telomere maintenance by telomerase" can't be equally called "DNA synthesis by telomerase"

Again, not sure but it may just be to make the "telomere maintenance" term names match nicely.

I seem to recall people (Eurie and Rob? guessing from def dbxrefs) saying that telomeres could be maintained either by synthesis or by recombination, which accounts for GO:0007004, GO:0032201 and GO:0000722. I also wonder if GO:0032201 should be is_a telomere maintenance via telomere lengthening GO:0010833?

_We have struggled to find evidence in human/mammalian non-cancer cells to support Alternative Lengthening of Telomeres (ALT) (which is a telomerase independent mechanism involving homologous recombination-mediated DNA replication). I am assuming that ALT is what this term GO:0032201 means/alludes to/is understood to mean or maybe is a more general term that includes ALT? (GO:0032201 telomere maintenance via semi-conservative replication: The process in which telomeric DNA is synthesized semi-conservatively by the conventional replication machinery and telomeric accessory factors as part of cell cycle DNA replication).

Wouldn't is_a mean that they are the same? - which they are not. Consequently, the two terms GO:0032201 telomere maintenance via semi-conservative replication &  and GO:0010833  telomere maintenance via telomere lengthening are fine as they stand. 

GO:0010833  telomere maintenance via telomere lengthening: Any process that contributes to the maintenance of proper telomeric length and structure by affecting and monitoring the activity of telomeric proteins and lengthening the telomeric DNA._

    isn't DNA strand elongation always a DNA biosynthetic process?

I think the strand elongation defs in GO are a bit simplistic; I would say that strand elongation is a process that always has DNA biosynthesis as a part, but not necessarily the only part. Certainly in the context of replication, lagging strand elongation involves primer synthesis (i.e. an RNA biosynthetic process) as well as DNA synthesis, and for both strands helicase and ligase activities are necessary, and not obviously covered by the GO:0022616 def. I would adjust the elongation defs to include the other activities, and then say elongation has_part synthesis.

_agree_

**Comment by NancyCampbell** [comment]
@tberardini 
sorry it was not clear.
I meant 
telomeric 3' overhang formation part_of telomere formation via telomerase
and a separate statement
telomerase RNA reverse transcriptase activity part_of telomeric 3' overhang formation (GO:0031860)

However, @mah11 commented on these. Ongoing discussion is in the post immediately prior to this.

**Comment by mah11** [comment]
Responses to just a few parts of Nancy's latest contribution ...

> how can one indicate that both components are needed together for telomerase activity and neither of them can perform the activity alone, in context of annotating an exisiting complex term?

In the case of telomerase, this could be done at the level of the ontology, by adding has_part links:

    telomerase activity GO:0003720 has_part GO:0000332 [template]
    and
    telomerase activity GO:0003720 has_part GO:0003721 [reverse transcriptase]

More generally, I'm not sure we have a reliable way of distinguishing at the level of annotation when a gene product makes an essential as opposed to dispensable contribution to an activity. Even using contributes_to doesn't quite say the contribution is essential, just that it makes a detectable difference. 

I wonder if there's anything relevant in LEGO?

> so if telomerase has other functions, for example some are TERC-independent i.e. TERT binds with RMRP and the TERT-RMRP complex has TERC-INDEPENDENT RNA-DEPENDENT RNA POLYMERASE ACTIVITY PMID 23061047 (figure 1 below) (review), the link can't be made, correct?

Not sure about this example, although at a quick glance it does seem inconsistent with making the link.

What I actually had in mind is that making the link would logically imply that the telomerase RT activity synthesizes only the 3' overhang. So if telomerase RT actually synthesizes more of the telomere than just the part that ends up as the 3' overhang, the link can't be made.

> We have struggled to find evidence in human/mammalian non-cancer cells to support Alternative Lengthening of Telomeres (ALT) (which is a telomerase independent mechanism involving homologous recombination-mediated DNA replication). I am assuming that ALT is what this term GO:0032201 means/alludes to/is understood to mean or maybe is a more general term that includes ALT?

That's another thing I don't know. (Judging from the ID I must have added the term years ago, but I think I did so using information from someone else, possibly Eurie and/or Rob.) But from your description ALT sounds more like GO:0000722, 'telomere maintenance via recombination'.

> Wouldn't is_a mean that they are the same? 

No, it would mean that one is more specific than the other, and that the more specific thing has all of the features of the less specific thing, plus one or more additional distinguishing features. If they were the same there would only need to be one term.

I thought from the part of the GO:0032201 definition that says "telomeric DNA is synthesized semi-conservatively by the conventional replication machinery" that it sounds an awful lot like a way of, well, lengthening a telomere.

**Comment by ValWood** [comment]
Re:

I seem to recall people (Eurie and Rob? guessing from def dbxrefs) saying that telomeres could be maintained either by synthesis or by recombination, which accounts for GO:0007004, GO:0032201 and GO:0000722. I also wonder if GO:0032201 should be is_a telomere maintenance via telomere lengthening GO:0010833?

We have struggled to find evidence in human/mammalian non-cancer cells to support Alternative Lengthening of Telomeres (ALT) (which is a telomerase independent mechanism involving homologous recombination-mediated DNA replication). 


===
I might be responsible for the term originally. I think you are correct and this is not a "normal" process. For sure, if you mutate telomerase, there are rare survivors where the chromosomes are circularized via recombination (which is usually prevented by the telomere proteins). I don't think that these mutants can survive long term.

This would help the clean up. Would require input from SGD

**Comment by ValWood** [comment]
and for sure, this is not replication. The telomeres are maintained only because the chromosome 
circularization prevents degradation.

**Comment by tberardini** [comment]
Reiterating my earlier comment: Great discussion. When it all settles down, please enumerate the changes you would like to see and I will implement.

**Comment by mah11** [comment]
I don't think I know enough nitty-gritty telomere biology to help more than I already have there (if it's been helpful at all). I do have some suggestions for DNA strand elongation:

GO:0022616 DNA strand elongation
new def: The DNA metabolic process in which an existing DNA strand is extended by activities including the addition of nucleotides to the 3' end of the strand. [GOC:mah, GOC:isa_complete]
add link has_part GO:0071897 DNA biosynthetic process

GO:0006271 DNA strand elongation involved in DNA replication
new def: The process in which an existing DNA strand is extended by activities including the addition of nucleotides to the 3' end of the strand, complementary to an existing template, as part of DNA replication. [GOC:mah, ISBN:071673706X, ISBN:0815316194]
add link has_part GO:0090592 DNA synthesis involved in DNA replication

GO:0006272 leading strand elongation
new def: The process in which an existing DNA strand is extended continuously in a 5' to 3' direction by activities including the addition of nucleotides to the 3' end of the strand, complementary to an existing template, as part of DNA replication. Leading strand elongation proceeds in the same direction as the replication fork. [GOC:mah, ISBN:071673706X, ISBN:0815316194]
add link has_part GO:0043137 DNA replication, removal of RNA primer
(keep existing link has_part GO:0006269 DNA replication, synthesis of RNA primer)

GO:0006273 lagging strand elongation
new def: The process in which an existing DNA strand is extended in a net 3' to 5' direction by activities including the addition of nucleotides to the 3' end of the strand, complementary to an existing template, as part of DNA replication. Lagging strand DNA elongation proceeds by discontinuous synthesis of short stretches of DNA, known as Okazaki fragments, from RNA primers; these fragments are then joined by DNA ligase. Although each segment of nascent DNA is synthesized in the 5' to 3' direction, the overall direction of lagging strand synthesis is 3' to 5', mirroring the progress of the replication fork. [GOC:mah, ISBN:071673706X, ISBN:0815316194]
add link has_part GO:0043137 DNA replication, removal of RNA primer
(keep existing link has_part GO:0006269 DNA replication, synthesis of RNA primer)

Also adjust text defs of regulation terms as needed to match.

Note: GO:0043137 should **not** be part_of GO:0006273, because each leading strand also has one primer that must be removed.


These have defs that don't really need changing, because they're just 'generic part involved in specific whole' without any of the bumf explaining what the part is:
- GO:1902296 DNA strand elongation involved in cell cycle DNA replication (but can add link has_part GO:1904860 DNA biosynthetic process involved in mitotic DNA replication)
- GO:1903459 mitotic DNA replication lagging strand elongation
- GO:1903460 mitotic DNA replication leading strand elongation

**Comment by pgaudet** [comment]
Hi @mah11 
This was a while ago. Can you confirm that your suggestions in the last comment are what needs to be done ? Thanks :) 

Pascale

**Comment by mah11** [comment]
I'm happy to confirm that the changes in the "strand elongation" area can be made. I'm less sure that sorts out everything about telomeres from earlier in the ticket, but I think at this point people can open new tickets for anything outstanding.

**Comment by pgaudet** [comment]
- [ ]  GO:0060382 regulation of DNA strand elongation
- [ ]  GO:0060383 positive regulation of DNA strand elongation

**Comment by pgaudet** [comment]
There are no regulation for lagging strand elongation, leading strand elongation, and DNA strand elongation involved in DNA replication.
